exercise-3-4-debugging-logging-getting-into-a-large-codebase-ahdsab-main/system_tests.py

```diff
@@ -0,0 +1,151 @@
+import unittest
+from enums import *
```

Comment:
Try to refrain from using import *.
There are things now in your code that are part of enum yet cannot be traced back to it easily when debugging (over importing explicitly).